data/recipes/gcp_cloud_resource_tree_offline.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gcp_cloud_resource_tree_offline",
+  "description": "Collects logs from a GCP project and create a resource tree",
```

Comment:
same as the other recipe, elaborate on what offline mode means

Reply:
Done